src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5333,9 +5756,9 @@ public override BoundNode VisitArrayInitialization(BoundArrayInitialization node
             return result;
         }
 
-        private void SetUnknownResultNullability()
+        private void SetUnknownResultNullability(BoundExpression expression)
```

Comment:
name and implementation have a discrepancy: unknown vs. not-null.

Reply:
I think Neal intended this method to use the word unknown to make it clear to the reader that it's being set to whatever the default happens to be, which in our analysis is NotNull. So I don't think it's appropriate to replace everywhere that's setting `NotNull`, because that's more explicit.

---
In reply to: [265757222](https://github.com/dotnet/roslyn/pull/34017#discussion_r265757222) [](ancestors = 265757222)

Reply:
I will change the `NullableFlowState.NotNull` to `default`, which is technically the same thing but communicates the intent more clearly.

---
In reply to: [265760228](https://github.com/dotnet/roslyn/pull/34017#discussion_r265760228) [](ancestors = 265760228,265757222)